actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -194,12 +194,12 @@ func (pb *Actor) CreateChannel(vmctx exec.VMContext, target address.Address, eol
 // target Redeem(200)          -> Payer: 1000, Target: 200, Channel: 800
 // target Close(500)           -> Payer: 1500, Target: 500, Channel: 0
 //
-func (pb *Actor) Redeem(vmctx exec.VMContext, payer address.Address, chid *types.ChannelID, amt *types.AttoFIL, validAt *types.BlockHeight, sig []byte) (uint8, error) {
```

Comment:
Currently the voucher parameter does nothing in this method aside from allowing the signature to validate. This will change very soon. I'll be sure to document the condition parameter once its are actually used.